lib/scan.ts

```diff
@@ -19,6 +19,17 @@ export function mergeEnvVarsIntoCredentials(
   options.password = options.password || process.env.SNYK_REGISTRY_PASSWORD;
 }
 
+function sanitizePlatform(options: Partial<PluginOptions>): void {
+  if (options.platform === undefined) {
+    options.platform = "linux/amd64";
+  } else if (
+    !options.platform.startsWith("linux/") ||
```

Comment:
It might be confusing to users that non-linux platforms are rejected as invalid - perhaps the error could, in this case, read that the platform is _unsupported_?

Reply:
Agreed 😄